hivemind_listener/transformers.py

```
@@ -0,0 +1,222 @@
+        A plugin of `priority` 1 will override any existing context keys and
+        will be the last to modify utterances`
+        """
+        return sorted(self.loaded_plugins.values(),
+                      key=lambda k: k.priority, reverse=True)
+
+    def shutdown(self):
+        for module in self.plugins:
+            try:
+                module.shutdown()
+            except:
+                pass
```

Ruff (0.8.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)